app/views/referrals/index.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for :breadcrumbs do %>
+  <%= render "/@shared/breadcrumbs/generic",
+        label: "Referrals", link: referrals_path, class: "active" %>
+<% end %>
+
+<% content_for :actions do %>
+  <%= render "/@shared/actions/placeholder" %>
+<% end %>
+
+<% content_for :page_heading do %>
+  <%= page_heading "Referrals", "",
+    subtitle: "Earn by referring friends",
+    icon: "fas fa-browser" %>
+<% end %>
+
+<main id="content" role="main" class="bg-light">
+  <div class="container space-2">
+    <div class="row justify-content-md-between">
+      <div class="col-md-6 mb-7 mb-md-0">
+        <div class="pr-md-4">
+          <!-- Title -->
+          <div class="mb-3">
+            <h2 class="h4">Share a link and earn a bonus</h2>
+            <p>Get a friend to join CodeFund as a publisher and earn 5% of the total revenue generated through their properties for 3 months.</p>
+          </div>
+          <!-- End Title -->
+
+          <!-- Clipboard Input -->
+          <form>
+            <div class="js-focus-state mb-2">
+              <div class="input-group">
+                <%= tag.input id: "referralLink", class: "form-control", value: invite_url(current_user.referral_code) %>
+                <div class="input-group-append">
+                  <a class="js-clipboard input-group-text" href="javascript:;"
+                    data-controller="copy-to-clipboard"
+                    data-content-target="#referralLink"
+                    data-class-change-target="#linkIcon"
+                    data-default-class="fas fa-clone"
+                    data-success-class="fas fa-check">
+                    <span id="linkIcon" class="fas fa-clone"></span>
+                  </a>
+                </div>
+              </div>
+            </div>
+            <small class="form-text text-muted">Copy or share your referral link with friends or followers</small>
+
+            <!-- AddToAny BEGIN -->
+            <div class="a2a_kit a2a_kit_size_32 a2a_default_style mt-3">
```

Comment:
For this to work you will need to set the meta stuff in the controller.

Reply:
I'll get you to walk me through that again.

Reply:
Are meta tags not already being set implicitly? https://github.com/gitcoinco/code_fund_ads/blob/master/app/controllers/application_controller.rb#L12